### PR TITLE
Form page

### PR DESCRIPTION
**Pull request by somoye123**
At HTML changes:
- Adding form, fieldset, labels, inputs, submit button, and the footer tags.
At CSS changes:
1. styling the form tag including:
2. the fieldset section
3. the labels and inputs
4. the footer section
To be as the given webpage for the project

### CODE REVIEW

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @somoye123  and @MahmoudBakr23   :handshake: 

### Good job so far :clap:  your project is looking good :medal_sports: 

### there are still some changes to get the project approved :point_down:

- In your Readme file there is non link to the live demo, please use http://raw.githack.com/ and update your readme file.
- The project should have a sign-in button:
##### Sign In Button
- [ ] It has a Sign In button at the top right corner ([link to page](https://accounts.intuit.com/signup.html))

Original:
Screenshot from 2020-06-03 13-12-40
Your project:
Screenshot from 2020-06-03 13-46-53

- The page should have the intuit logo and products:
##### Intuit logo
- [ ] It shows the Intuit logo above the form ([link to page](https://accounts.intuit.com/signup.html))
- [ ] The intuit logo is an embedded image that is horizontally centered ([link to page](https://accounts.intuit.com/signup.html))

##### Intuit products
- [ ] The products row is under the Intuit logo and above the form ([link to page](https://accounts.intuit.com/signup.html))
- [ ] It shows the Mint logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))
- [ ] It shows the Quickbooks logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))
- [ ] It shows the Turbotax logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))

Original:
Screenshot from 2020-06-03 13-15-02
Your Project:
Screenshot from 2020-06-03 13-46-53
- Having a footer is required:
##### Footer
- [ ] It contains copyright and address information ([link to page](https://accounts.intuit.com/signup.html))
Original:
Screenshot from 2020-06-03 13-21-57
Your Project: You don't have a footer.
### Check the comments for more details.
### Please implement the required changes and ask for another code review.

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @somoye123  and @MahmoudBakr23   :handshake: 

### Good job so far :clap:  you implemented what I requested earlier and your project is looking good :medal_sports: 

### there are still some changes that come up to get the project approved :point_down:
- Always passe the HTML file through the validators before a pull request:
Screenshot from 2020-06-04 01-03-33
Please fix these validations.
- there are some issues with the images:
your project:
Screenshot from 2020-06-04 01-04-21
Check the comment I will add some suggestions on how to fix the images.
## please implement the changes and ask for another code review. :+1:

File: index.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      </div>
+      <button type="submit" name="submit"><i class="fas fa-lock"></i> Sign In</button>
+    </div>
+    <div class="logos">
+      <img src="/images/intuit-logo-removebg-preview.png" alt="intuit-logo">
+    </div>
+    <ul class="logos-list">
+      <li><img src="/images/mint-logo-removebg-preview.png"></li> <p>mint</p>
+      <li><img src="/images/logo-qbo.png"></li> <p>quickbooks</p>
+      <li><img src="/images/png-transparent-turbotax-computer-software-tax-return-intuit-type-miscellaneous-company-trademark-removebg-preview.png"></li> <p>turbotax</p>
```
Comment (HADDADSOHAIB) [inline index.html]: Use an absolute path in the src attribute, it should be src="**.**/images/.......".

**Comment by andresporras** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

**Review by andresporras — CHANGES_REQUESTED** [review]
### Status: Change Required :recycle:
**Hello @somoye123 and @MahmoudBakr23** 
You did a great implementing some improvements, your images now have an absolute path, your page looks really close to the original  :clap:
There are still some improvements to do:

- [ ]  Check HTML markup with this validator https://validator.w3.org/#validate_by_upload

By now the validator shows some errors in your code.

### OPTIONAL

- [ ] it's recommended to use the main tag for the dominant content of the body of the document
- [ ] it's recommended to use nav tag for the major navigation links

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Comment by somoye123** [comment]
We used the validator and we found no errors.
Please check the index.html file within the repo directory instead of the live demo

Screenshot (181)

**Comment by somoye123** [comment]
@andresporras

**Review by lucasmazo32 — APPROVED** [review]
# Project approved :rocket:

#### Great job!

#### Your project looks great!

### You can go ahead and merge this pull request!

### Happy coding! :computer:

## Lucas Mazo

**Pull request merged**